server/commands.py

```diff
@@ -181,6 +181,49 @@ def ooc_cmd_pos(client, arg):
         client.area.broadcast_evidence_list()
         client.send_host_message('Position changed.')   
 
+def ooc_cmd_forcepos(client, arg):
+    if not client.is_cm and not client.is_mod:
+        raise ClientError('You must be authorized to do that.')
+
+    args = arg.split()
+
+    if len(args) < 1:
+        raise ArgumentError(
+            'Not enough arguments. Use /forcepos <pos> <target>. Target should be ID, OOC-name or char-name. Use /getarea for getting info like "[ID] char-name".')
```

Comment:
You did not document the ability to invite all players if the <target> argument was not specified. Also, `pos` does not seem to be checked early for validity.

Reply:
It's checked by the ClientError bit, actually. Seems to be working out fine.